app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/model/DummyCategorizedPicture.kt

```diff
@@ -0,0 +1,23 @@
+package com.github.HumanLearning2021.HumanLearningApp.model
+
+import android.app.Activity
+import android.widget.ImageView
+import com.github.HumanLearning2021.HumanLearningApp.R
+import java.lang.IllegalArgumentException
+
+/**
+ * A picture part of the dummy data set. Can be of any of the following categories: "fork", "knife", "spoon"
+ */
+data class DummyCategorizedPicture(override val category: Category) : CategorizedPicture {
+
+    override fun displayOn(activity: Activity, imageView: ImageView) {
+        if(category !is DummyCategory) throw IllegalArgumentException("provide a dummy category to the class constructor")
```

Comment:
This invariant can be checked in the primary constructor, but it's good enough right now